en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Release policy
+summary: Outline the release lifecycle and support commitment for Silverstripe CMS
+icon: code-branch
+---
+
+# Silverstripe CMS release policy
+
+This document outlines the release policy for Silverstripe CMS. It applies to all [commercially supported modules](/project_governance/supported_modules/).
+
+Community modules are not covered by this policy. Modules in the silverstripe GitHub organisation that are not commercially supported are updated on a best effort basis.
+
+> [!NOTE]
+> Refer to our [definition of public API](/project_governance/public_api/).
+
+The purpose of this policy is to provide clear guidance on the types of releases, their cadence, scope, and support lifecycle, ensuring predictability and stability for our users and developers.
+
+## Release types and cadence
+
+Silverstripe CMS adheres to [Semantic versioning](https://semver.org/) for its release philosophy.
+
+### Major releases `(x.0.0)`
+
+Major releases include architectural updates, allow breaking changes and major dependency changes.
```

Comment:
```suggestion
The main function of major releases is to ship breaking changes that can not be shipped in minor releases and update third-party dependencies. These changes are primarily aimed at making sure the major releases can be supported for the full duration of its lifecycle.
```
From the existing policy - updated a bit to include third party dependencies which comes from the list "Most of the changes shipped in a major release will:"

Reply:
Done, though added to the "purpose" bolded bullet as I've gotten rid of the blurbs